hack/gen-swagger-doc/deploy.sh

```diff
@@ -38,7 +38,8 @@ find * -type d -regex "^v[0-9.]*" \
 git config --global user.email "travis@travis-ci.org"
 git config --global user.name "Travis CI"
 
-if git status --porcelain | grep --quiet "^ [AM]"; then
+# NOTE: exclude index.html from match, becasue it is static except commit hash.
```

Comment:
I couldn't come with something better. If you have any better idea, I will be glad to hear!